### PR TITLE
get-flash-videos: Add formula for get-flash-videos

### PR DESCRIPTION
**Pull request by onlynone**

### CODE REVIEW

File: Library/Formula/get-flash-videos.rb [hunk 1]
```diff
@@ -0,0 +1,16 @@
+class GetFlashVideos < Formula
+  homepage "https://code.google.com/p/get-flash-videos"
+  url "http://get-flash-videos.googlecode.com/files/combined-get_flash_videos-1.24", :using => :nounzip
+  version "1.24"
+  sha256 "e41da715f817bfa1a65ae82cc4f5a82268b6e72a7f4d90ffabf89b3a522cbb91"
+
+  def install
+    mv "combined-get_flash_videos-1.24", "get_flash_videos"
```
Comment (mistydemeo) [inline Library/Formula/get-flash-videos.rb]: You can do this in one step:

```ruby
bin.install "combined-get_flash_videos-1.24" => "get_flash_videos"
```

**Comment by mistydemeo** [comment]
A couple of comments from `brew audit`:

```

==> audit problems
get-flash-videos:
 * Google Code homepage should end with a slash (URL is https://code.google.com/p/get-flash-videos).
 * Use https:// URLs for downloads from Google Code (url is http://get-flash-videos.googlecode.com/files/combined-get_flash_videos-1.24).
```

**Comment by mistydemeo** [comment]
This seems to be maintained on Github now: https://github.com/monsieurvideo/get-flash-videos Is it worth using that as the homepage/URL?

**Comment by onlynone** [comment]
Possibly, I didn't read up on how to build the combined executable from source that has all the dependencies included. It appears that the project is developed on github, but they still post their releases on code.google.com.

File: Library/Formula/get-flash-videos.rb (hunk 1 above)
Comment (onlynone) [inline Library/Formula/get-flash-videos.rb]: I knew there had to be a way! I took forever trying to figure out how before giving up.

**Comment by mistydemeo** [comment]
Sure, maybe download the combined script from code.google.com while pointing the homepage at Github?

**Comment by onlynone** [comment]
Anything more I can do to get this merged?

**Comment by mikemcquaid** [comment]
Thanks for your contribution to Homebrew! Without people like you submitting PRs we couldn't run this project. You rock!

For future reference the preferred commit message format for simple version updates is `get-flash-videos 1.2.4`.

**Pull request closed without merging**